python/py-mdtraj/Portfile

```diff
@@ -29,8 +29,13 @@ if {${name} ne ${subport}} {
     depends_build-append    port:py${python.version}-cython \
                             port:py${python.version}-setuptools
```

Comment:
you should remove it from `depends_build`; if you specify it as a `lib` dependency it means: both at build and runtime